assets/img/icon-sprite.svg

```diff
@@ -1 +1 @@
14656 C15.0474453,12.578881 15.473927,12.1445339 16,12.1445339 C16.526073,12.1445339 16.9525547,12.578881 16.9525547,13.114656 L16.9525547,14.2429415 Z" id="stumbleupon-Fill-1" sketch:type="MSShapeGroup" mask="url(#stumbleupon-mask-2)"/> </symbol><symbol viewBox="0 0 32 32" id="icon-tumblr"> <path d="M23.852762,25.5589268 C23.2579209,25.8427189 22.1195458,26.089634 21.2697728,26.1120809 C18.7092304,26.1810247 18.2121934,24.3131275 18.1897466,22.9566974 L18.1897466,12.9951133 L24.6159544,12.9951133 L24.6159544,8.15140729 L18.2137967,8.15140729 L18.2137967,0 L13.5256152,0 C13.4486546,0 13.3139736,0.0673405008 13.2963368,0.238898443 C13.0221648,2.73370367 11.8533261,7.11243957 7,8.86168924 L7,12.9951133 L10.2387574,12.9951133 L10.2387574,23.4521311 C10.2387574,27.0307977 12.8794671,32.1166089 19.8508122,31.9979613 C22.2013164,31.9578777 24.8131658,30.9718204 25.3919735,30.1236507 L23.852762,25.5589268"/> </symbol><symbol viewBox="0 0 43 32" id="icon-twitter"> <path d="M36.575 5.229c1.756-0.952 3.105-2.46 3.74-4.257-1.644 0.882-3.464 1.522-5.402 1.867-1.551-1.495-3.762-2.429-6.209-2.429-4.697 0-8.506 3.445-8.506 7.694 0 0.603 0.075 1.19 0.22 1.753-7.069-0.321-13.337-3.384-17.532-8.039-0.732 1.136-1.152 2.458-1.152 3.868 0 2.669 1.502 5.024 3.784 6.404-1.394-0.040-2.706-0.386-3.853-0.962-0.001 0.032-0.001 0.064-0.001 0.097 0 3.728 2.932 6.837 6.823 7.544-0.714 0.176-1.465 0.27-2.241 0.27-0.548 0-1.081-0.048-1.6-0.138 1.083 3.057 4.224 5.281 7.946 5.343-2.911 2.064-6.579 3.294-10.564 3.294-0.687 0-1.364-0.036-2.029-0.108 3.764 2.183 8.235 3.457 13.039 3.457 15.646 0 24.202-11.724 24.202-21.891 0-0.334-0.008-0.665-0.025-0.995 1.662-1.085 3.104-2.439 4.244-3.982-1.525 0.612-3.165 1.025-4.885 1.211z"/> </symbol><symbol viewBox="0 0 32 32" id="icon-youtube"> <path d="M31.6634051,8.8527593 C31.6634051,8.8527593 31.3509198,6.64879843 30.3919217,5.67824658 C29.1757339,4.40441487 27.8125088,4.39809002 27.1873503,4.32353816 C22.7118278,4 15.9983092,4 15.9983092,4 L15.984407,4 C15.984407,4 9.27104501,4 4.79536595,4.32353816 C4.17017613,4.39809002 2.80745205,4.40441487 1.59082583,5.67824658 C0.631890411,6.64879843 0.319843444,8.8527593 0.319843444,8.8527593 C0.319843444,8.8527593 0,11.4409393 0,14.0290881 L0,16.4554834 C0,19.0436008 0.319843444,21.6317495 0.319843444,21.6317495 C0.319843444,21.6317495 0.631890411,23.8357417 1.59082583,24.8062935 C2.80745205,26.0801566 4.40557339,26.0398591 5.11736986,26.1733699 C7.67602348,26.4187241 15.9913894,26.4946536 15.9913894,26.4946536 C15.9913894,26.4946536 22.7118278,26.4845401 27.1873503,26.1610333 C27.8125088,26.0864501 29.1757339,26.0801566 30.3919217,24.8062935 C31.3509198,23.8357417 31.6634051,21.6317495 31.6634051,21.6317495 C31.6634051,21.6317495 31.9827789,19.0436008 31.9827789,16.4554834 L31.9827789,14.0290881 C31.9827789,11.4409393 31.6634051,8.8527593 31.6634051,8.8527593 Z M12.6895342,19.39582 L12.6880626,10.4095186 L21.3299413,14.9183249 L12.6895342,19.39582 Z" id="youtube-Imported-Layers"/> </symbol></svg>
```

Comment:
This file shouldn't have a diff. I believe this is related to https://github.com/bigcommerce/cornerstone/issues/1246. We need to remove the "mask" and " sketch:type" attributes from instagram.svg and stumbleupon.svg. These are included in the svg sprite and causing issues when loading in certain browsers. I believe there were some manual changes to the sprite in an earlier release.

Reply:
Ah I see, I'll see what I can do.